package cl.coopeuch.desafio.exceptions;

public class NotFoundException  extends RuntimeException {

}